.kitchen.yml

```
@@ -0,0 +1,16 @@
+---
+verifier:
+  name: inspec
+
+suites:
+  - name: default
+    run_list:
+       - recipe[osl-scponly]
+  - name: _scponly
```

Why are you adding an underscore here and on the suite below?
```suggestion
  - name: scponly
```

---

I thought that was convention for test recipes.

---

No, that's only for recipes that shouldn't be directly referenced and are included by other recipes.